### ISSUE TITLE
[Server thread/ERROR]: Could not pass event InventoryOpenEvent to Confiscate v1.0.1

### ISSUE THREAD

**Issue by n810** [kangarko/Confiscate#2]
Hi.

I purchased the plugin and tested it on my server against my own player and it worked perfrctly.

Now I see in my logs constant errors [Server thread/ERROR]: Could not pass event InventoryOpenEvent to Confiscate v1.0.1 and nothing is being confiscated, even though I am sure there should be.

I have not modified the default configuration in any way and I am running 
Minecraft server version 1.11.2 with CraftBukkit version git-Spigot-283de8b-4316a8c (MC: 1.11.2) (Implementing API version 1.11.2-R0.1-SNAPSHOT)

As far as I can tell I have no other plugins that might cause a problem.

Copy of the log output related to this is provided.
http://pastebin.com/i0ChTXJc

**Comment by kangarko** [comment]
Hello,

thank you for purchasing Confiscate. I am terribly sorry for any issues caused. I have a server myself and I know how annoying it is to find stack traces in the console, especially if you just purchased a new plugin. 

I have looked at the cause of this problem, and it lies in the PowerNBT library. We use this library to put an invisible tag onto the confiscated item, so when you have copied it from the chest, it won't be confiscated again.

There is nothing on my end I can do to directly fix it, but I have made it so that no error is printed, and such items are considered as being without any tag. **A new release will be available within 1-2 hours**.

Thank you.
